crawler/config/centre.py

```diff
@@ -29,6 +29,13 @@
 CENTRE_REGEX_BARCODE = r"^[\W_]*([\w-]*)_([A-Z]\d{0,1}\d)[\W_]*$"
 CENTRE_DIR_BACKUPS = "data/backups"
 CENTRE_REGEX_SFTP_FILE_HERON = r"sanger_report_(\d{6}_\d{4}).*\.csv$"
+AP_SURVEILLANCE_1 = r"^AP-rna-\d+\.csv$"
```

Comment:
I think the convention we're aiming at is to promote easily finding variables with a quick glance and potentially sorting them so my suggestion would be something like: `REGEX_SURVEILLANCE_AP_1` etc.